config/CConfig.py

```diff
@@ -60,6 +60,7 @@
 import os, sys, platform, shlex, subprocess
 import colorama as col
 import pypandoc
+from RobotResults2RQM import VERSION
```

Comment:
Hi Ngoan,
this is really a good idea. This makes maintenance easier and allows an interface to the application which use the package.
Thank you,
 Thomas